hc/api/models.py

```diff
@@ -143,7 +143,7 @@ def notify(self, check):
                 "icon_url": "https://healthchecks.io/static/img/logo@2x.png"
             }
 
-            r = requests.post(self.value, json=payload)
+            r = requests.post(self.value, json=payload, timeout=5)
```

Comment:
the webhook request has a timeout of 5
these other external API calls should also have the same timeout to prevent accidental stalls